modules/server-endpoints/language-server/pom.xml

```diff
@@ -56,38 +55,32 @@
         <dependency>
             <groupId>javax.websocket</groupId>
             <artifactId>javax.websocket-api</artifactId>
-            <scope>provided</scope>
         </dependency>
 
         <dependency>
             <groupId>org.wso2.carbon.identity.framework</groupId>
             <artifactId>org.wso2.carbon.identity.application.authentication.framework</artifactId>
-            <scope>provided</scope>
         </dependency>
 
         <dependency>
             <groupId>org.wso2.carbon.identity.framework</groupId>
             <artifactId>org.wso2.carbon.identity.functions.library.mgt</artifactId>
-            <scope>provided</scope>
         </dependency>
 
         <dependency>
             <groupId>org.wso2.carbon.identity.developer</groupId>
             <artifactId>org.wso2.carbon.identity.developer.java-agent</artifactId>
-            <scope>provided</scope>
+
```

Comment:
We should not have hard dependency. May be we can refactor and have a common module to keep common classes.